ci/citest/acceptance-test/multibox/10.0.100.16-vdc-executor-qemu/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+export ENV_ROOTDIR="$(cd "$(dirname $(readlink -f "$0"))/.." && pwd -P)"
+export NODE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+TMP_ROOT="${NODE_DIR}/tmp_root"
+
+. "${ENV_ROOTDIR}/config.source"
+. "${NODE_DIR}/vmspec.conf"
+. "${ENV_ROOTDIR}/ind-steps/common.source"
+
+IND_STEPS=(
+    "box"
+    "ssh"
+    "hosts"
+    "disable-firewalld"
+    "epel"
+    "qemu"
+    "kvm"
+    "ncat"
+    "bridge-utils"
+    "mesosphere-repo"
+    "mesos-agent"
+    "vdc-images"
```

Comment:
The reason I decided to include them is because the driver does not support compressed types right now and it took awhile to download them image. The "local image server" also seems like it doesn't translate as well to qemu, the images doesn't really have a template and I'm not so sure about parsing out a "local" keyword just for that sake, instead just supplying the full url (local ip) in the son file feels more natural to me.